setup.py

```diff
@@ -473,6 +479,40 @@ def run(self):
 
             self.copy_file(export_lib, target_lib)
 
+    def build_extension(self, ext):
+        if ext.name == 'torch._C':
+            cmake_build_dir = os.path.join(self.build_lib, 'torch')
+            try:
+                os.makedirs(cmake_build_dir)
+            except OSError as e:
+                if e.errno != errno.EEXIST:
+                    raise
+
+            cmake_cmd = [
```

Comment:
Any chance we could put this into `tools/build_pytorch_libs.sh` instead? Eventually we want this built as a part of the root-level cmake (and therefore remove build_pytorch_libs.sh) and having it here will make that more difficult.